src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOneIntegrationTest.java

```diff
@@ -925,13 +642,77 @@ public void inConditionForUUIDField() {
   @Nested
   @Order(3)
   @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+  class FindOneWithDottedPaths {
```

Comment:
And here are actual tests: for now just 2 tests. When add support for escaping will add more, covering ampersands -- right now just matches against now legal dotted paths